src/main/java/org/iitbact/cc/entities/FacilityContact.java

```diff
@@ -44,7 +44,6 @@
 	private static final long serialVersionUID = 1L;
 
 	@Id
-	@Column(name="facility_id")
 	private Integer facilityId;
```

Comment:
Are you sure about the column name? It is facility_id everywhere.

Reply:
I was seeing a warning for using @Column together with @JoinColumn below. I found that this did not change any functionality, but please put it back if it did.

Reply:
The column name is facility_id, so this might break something.

Entity won't work without an Id field so I kept this. Maybe we can remove this field altogether, I am not sure.

Reply:
I actually see an error, not a warning.

In attribute 'facilityId', a column is specified, but it is mapped by a relationship. IDs that are mapped by a relationship should not specify a column.